src/lightning/app/storage/copier.py

```diff
@@ -103,15 +104,18 @@ def _find_matching_path(work, request: _GetRequest) -> Optional["lightning.app.s
             return candidate
 
 
-def _copy_files(source_path: pathlib.Path, destination_path: pathlib.Path) -> None:
+def _copy_files(
+    source_path: pathlib.Path, destination_path: pathlib.Path, fs: Optional[AbstractFileSystem] = None
```

Comment:
This isn't being used anywhere. The filesystem you added does not have that type. Probably from another PR?

Reply:
This is done to avoid re-creating the S3FS filesystem that is time consuming.

Reply:
> The filesystem you added does not have that type
Right now, it is used only in the codebase and it should be an `AbstractFileSystem`.